reimplementation.py

```diff
@@ -519,13 +751,154 @@ def create_information_on_classes_to_evaluate(mixture_classes_in_single_cell_typ
         for i in set(labels):
             y_combi[np.where(np.array(y_mixtures) == i), i_combination] = 1
 
-    return mixture_classes_in_classes_to_evaluate, np.append(y_mixtures_matrix, y_combi, axis=1)
+    return mixture_classes_in_classes_to_evaluate, \
+           np.append(y_mixtures_matrix, y_combi, axis=1)
+
+
+def getNumeric(prompt):
```

Comment:
kunnen deze en onderstaande functies in een aparte file? Is volgens mij iets dat helpt bij de data op orde krijgen (eenmalig)?